Makefile

```diff
@@ -1,9 +1,7 @@
-.PHONY: build test run clean stop check-style fix-style run-unit emojis help package-ci update-dependencies
+.PHONY: build dist test run clean stop check-style fix-style run-unit emojis help update-dependencies
 
 BUILD_SERVER_DIR = ../mattermost-server
-BUILD_WEBAPP_DIR = ../mattermost-webapp
-MM_UTILITIES_DIR = ../mattermost-utilities
-EMOJI_TOOLS_DIR = ./build/emoji
+
```

Comment:
From what I am understanding we will no longer build the packages from CI and instead compile from the dist folder?

Reply:
The change there is that `make dist` will be used everywhere instead of some places using `make package-ci` and others using `make build` directly. Previously, `package-ci` also made npm install dependencies using `npm ci` instead of `npm install`, but I made it so that that was done automatically based on an environment variable set by CI rather than using different Makefile targets